src/2018/transitioning/modules/macros.md

```diff
@@ -43,4 +43,4 @@ This moves `macro_rules` macros to be a bit closer to other kinds of items.
 ## More details
 
 This only works for `extern crate`; if you write a macro in the same crate,
-nothing changes.
\ No newline at end of file
+nothing changes from Rust 2015.
```

Comment:
I thought use_extern_macros permits using macros from within a crate as well. I seem to recall recent discussions toward this being a change we're making, but I'm not completely up to date...